src/sentry/feedback/endpoints/feedback_ingest.py

```diff
@@ -42,6 +42,9 @@ class FeedbackValidator(serializers.Serializer):
     request = serializers.JSONField(required=False)
     tags = serializers.JSONField(required=False)
     user = serializers.JSONField(required=False)
+    contexts = serializers.JSONField(required=False)
+    BrowserContext = serializers.JSONField(required=False)
+    DeviceContext = serializers.JSONField(required=False)
```

Comment:
you can have these be snake case in the code by using the `source` field, like here: https://github.com/getsentry/sentry/blob/47fe7989a75c5015c6080a1e29c4e025bea45139/src/sentry/scim/endpoints/utils.py#L60